### PR TITLE
Display macros under the Macros tab when creating a new desk, too

### PR DESCRIPTION
**Pull request by plamut**
Resolves [SD-2543](https://dev.sourcefabric.org/browse/SD-2543).

### CODE REVIEW

File: client/spec/desks_management_spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+var desks = require('./helpers/desks');
+
+describe('desks management', function () {
+
+    beforeEach(function() {
+        desks.openDesksSettings();
+    });
+
+    it('lists macros under the Macro tab for new desks', function () {
+        desks.openDesksSettings();
```
Comment (sivakuna-aap) [inline client/spec/desks_management_spec.js]: This is not necessary as `desks.openDesksSettings();` is covered by `beforeEach()` already.

**Comment by sivakuna-aap** [comment]
:+1: except the minor comment

File: client/spec/desks_management_spec.js (hunk 1 above)
Comment (plamut) [inline client/spec/desks_management_spec.js]: Meh, copy/paste, thanks. :)

**Comment by akintolga** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**